W-22395996 added other supported assets (#91)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/index.adoc

include::_attributes.adoc[]
endif::[]

> Anypoint API Experience Hub is a portal management platform and digital experience for APIs and agentic AI, helping organizations build vibrant AI ecosystems within and outside the enterprise. 

Reviewer: @JennyHajee 
```suggestion
Anypoint API Experience Hub empowers organizations to build thriving API and AI ecosystems by providing a unified portal experience for discovering and consuming APIs and agentic services, whether for internal teams, partners, or external developers.
```

. Connect to a Salesforce instance.
+
The instance can be a new or existing Salesforce organization. To connect, specify the environment type and enter the custom Salesforce domain if you have one, log in, allow the permissions, and confirm the account.
. Set up your portal.
+
Once you connect to Salesforce, choose how to set up the portal and style the portal's out of the box pages with logos, fonts, color schemes, and more.
. Manage your portal.
+
Manage the catalog by adding services from Exchange and setting the visibility for the versions. Manage users by approving access requests from guest users.
. Publish the portal.
+
Publish the portal to make it available to your community.

== Portal Pages

> Each API Experience Hub portal includes a Home page, a Services page, and a Service Details page. The portal administrator can customize each page to fit the organization's needs. For more information, see xref:setting-up-the-api-portal.adoc[].

Reviewer: @Kabilan-s-cx 
```suggestion
Each API Experience Hub portal includes a Home, APIs page, and an API Details page. The portal administrator can customize each page to fit the organization's needs. For more information, see xref:setting-up-the-api-portal.adoc[].
```

//The following sample portal shows the Home page: 

//image::aeh-home-page.png[Home page of a portal]

The portal includes these template pages:

* Home
+

> The Home page includes the banner image, portal title, Carousel, and navigation to services by type and serves as the landing page of the portal. The Carousel offers consumers a preview of the most recent services added to the portal. Each card contains the service name, icon, type, and description. When selected, developers can view all the information about a service on the service details page.  

Reviewer: @Kabilan-s-cx 
```suggestion
The Home page includes the banner image, portal title, Carousel, and navigation to services by type and serves as the landing page of the portal. The Carousel offers consumers a preview of the most recent services added to the portal. Each card contains the service name, icon, type, and description. When selected, developers can view all the information about a service on the API Details page.  
```

> * Services

Reviewer: @Kabilan-s-cx 
```suggestion
* APIs
```

+

> The Services page contains a catalog of all the services in the portal. Developers can search and filter the catalog. When selected, each card in the catalog shows the service details page. 

Reviewer: @Kabilan-s-cx 

```suggestion
The APIs page contains a catalog of all the services in the portal. Developers can search and filter the catalog. When selected, each card in the catalog shows the API Details page. 
```

> * Services Details 

Reviewer: ```suggestion
* API Details 
```

+

> The services Details page shows the metadata about the service, including the name, description, type, version, lifecycle states, and the email of the contact for the service. The details page includes the overview and the service documentation. The overview panel reflects the overview page created by the admin of the service in Anypoint Exchange. The documentation panel includes an interactive console for developers to read the service documentation and test the service. Developers can read the summary, methods, endpoints, and authorization sections as defined in the API specification. 

Reviewer: ```suggestion
The API Details page shows the metadata about the service, including the name, description, type, version, lifecycle states, and the email of the contact for the service. The details page includes the overview and the service documentation. The overview panel reflects the overview page created by the admin of the service in Anypoint Exchange. The documentation panel includes an interactive console for developers to read the service documentation and test the service. Developers can read the summary, methods, endpoints, and authorization sections as defined in the API specification. 
```
